v1/messages/src/requestHandlers.ts

```diff
@@ -4,13 +4,14 @@ interface MessagePayload {
   message: string;
   token: string;
 }
+const validURLCharacters = /^[A-Za-z0-9\-._~:/?#[\]@!$&'()*+,;=]*$/;
```

Comment:
If you're going to use regex for this, wouldn't it be better to just restrict the the authID to only being numeric digits?

```suggestion
const digitsOnlyRegex = /^\d+$/;
```